rails.gemspec

```diff
@@ -27,6 +27,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'activejob',     version
   s.add_dependency 'railties',      version
 
-  s.add_dependency 'bundler',         '>= 1.3.0', '< 2.0'
+  s.add_dependency 'bundler',         '>= 1.9.0', '< 2.0'
```

Comment:
I think we don't need to require this version just because an generated Gemfile. Do you have any specific reason to bump the requirement besides the Gemfile change?